pkg/steps/multi_stage.go

```diff
@@ -45,10 +45,15 @@ const (
 	CliMountPath = "/cli"
 	// CliEnv if the env we use to expose the path to the cli
 	CliEnv = "CLI_DIR"
-	// CommandPrefix is the prefix we add to a user's commands
-	CommandPrefix = "#!/bin/bash\nset -eu\n"
+	// CommandScriptFilename is the filename for the command script
+	CommandScriptFilename = "commands.sh"
```

Comment:
Can we get this from the step registry? Will non-shell files be unhappy about being called `.sh` ? Does `commands.sh` run in Python fine?

Reply:
As long as the shebang points to a valid executable, the filename doesn't matter.